### PR TITLE
 settings for ACE are moved to cba_settings.sqf

### LINKED ISSUE #169: [ACE related] Getting killed instantly instead of falling unconscious after Ace update Dec 21st
After the Ace update from Dec. 21st in many cases of being critically injured (where you would previously fall unconscious) you die instantly now (the "MajorJack was killed" way). 

This happened repeatedly from ordinary handweapon fire (no big guns involved).

I guess this might have to be fixed by the ace guys themselves, wanted to report it anyways.

### PR DESCRIPTION
**Pull request by IrLED**
- settings for ACE are moved to cba_settings.sqf
- code scope fix
- removal of hardcoded dependency to CBA. Now should work on vanilla

Fixes #169

### CODE REVIEW

File: AntistasiOfficial.Altis/AAFeconomics.sqf [hunk 1]
```diff
@@ -41,8 +41,8 @@ if (_resourcesAAF > 5000) then{
 						_resourcesAAF = _resourcesAAF - (5000*_multiplicador);
 					};
 				};
-			} forEach antenasMuertas; */ //Stef disabled repair radiotower, the mission is buggy, reduntant and pointless
-		};
+			} forEach antenasMuertas;
+		}; */ //Stef disabled repair radiotower, the mission is buggy, reduntant and pointless
```
Comment (IrLED) [inline AntistasiOfficial.Altis/AAFeconomics.sqf]: @StefArma 
be careful next time, that broke the code scope

File: AntistasiOfficial.Altis/mission.sqm [hunk 2]
```diff
@@ -34,7 +34,6 @@ addons[]=
 	"A3_Structures_F_Mil_Flags",
 	"A3_Structures_F_Civ_InfoBoards",
 	"A3_Structures_F_Civ_Camping",
-	"cba_xeh",
```
Comment (IrLED) [inline AntistasiOfficial.Altis/mission.sqm]: That makes mission vanilla friendly

**Comment by Sparker95** [comment]
As @IrLED said, this should make the ace settings work 'out of the box' since they are moved to CBA now. So a user won't need to go to 'addon options' and set it up.

**Comment by IrLED** [comment]
Mission started ok with vanilla. Might be a good candidate for Steam release.

File: AntistasiOfficial.Altis/AAFeconomics.sqf (hunk 1 above)
Comment (StefArma) [reply AntistasiOfficial.Altis/AAFeconomics.sqf]: Damn! sorry, thanks!

File: AntistasiOfficial.Altis/mission.sqm (hunk 2 above)
Comment (StefArma) [reply AntistasiOfficial.Altis/mission.sqm]: Fantastic!

**Comment by IrLED** [comment]
@StefArma 
regarding Steam let's continue here #175

**Comment by Sparker95** [comment]
As @AlexTriada said in a private talk, adding settings to cba_settings.sqf locks them, and they can't be changed by server options any more, creating troubles for server owners.
We should consider removing it.
https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System

**Pull request merged**